lib/config/constants/constants.go

```diff
@@ -23,10 +23,16 @@ const (
 
 	TemporaryTableTTL = 6 * time.Hour
 
-	DBZPostgresFormat    = "debezium.postgres"
+	// DBZPostgresFormat - deprecated - Use `DBZRelationalFormat` instead
+	DBZPostgresFormat = "debezium.postgres"
+	// DBZPostgresAltFormat - deprecated - Use `DBZRelationalFormat` instead
 	DBZPostgresAltFormat = "debezium.postgres.wal2json"
-	DBZMongoFormat       = "debezium.mongodb"
-	DBZMySQLFormat       = "debezium.mysql"
+	// DBZMongoFormat - deprecated - Use `DBZRelationalFormat` instead
```

Comment:
Is this comment wrong? The Mongo format isn't superseded by the relational format is it?

Reply:
Sorry - yes this comment is wrong re: Mongo